scripts/run-bundled-codegen.sh

```diff
@@ -20,16 +20,14 @@ remove_existing_apollo() {
 extract_cli() {
   tar xzf "${SCRIPT_DIR}"/apollo.tar.gz -C "${SCRIPT_DIR}"
   
-  # Get just the SHASUM rather than the SHASUM + path
-  ARRAY=($SHASUM)
-  echo "${ARRAY[0]}" | tee "${SHASUM_FILE}"
+  echo "${SHASUM}" | tee "${SHASUM_FILE}"
 }
 
 validate_codegen_and_extract_if_needed() {
   # Make sure the SHASUM matches the release for this version
   EXPECTED_SHASUM="13febaa462e56679099d81502d530e16c3ddf1c6c2db06abe3822c0ef79fb9d2"
 
-  if [[ ${SHASUM} = ${EXPECTED_SHASUM}* ]]; then
+  if [[ ${SHASUM} = ${EXPECTED_SHASUM} ]]; then
```

Comment:
The asterisk here (and below) is intentional - it's designed to match `0.16.0` produced versions of the `.shasum` file that also have the file name in there as well.

Reply:
Ah got ya.